frontend/src/components/Landing/index.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 
+import SignInButton from './signin-button';
+import Card from 'react-bootstrap/Card';
```

Comment:
According to the style guide, you must include the ".js" suffix when importing files: https://google.github.io/styleguide/jsguide.html#es-module-imports